GiniVision/Classes/Core/GiniConfiguration.swift

```diff
@@ -616,11 +574,7 @@ import UIKit
      - Warning: Deprecated, use the ginivision.review.bottom localized string instead
 
      */
-    @objc public var reviewTextBottom =
-        NSLocalizedStringPreferred("ginivision.review.bottom",
-                                   comment: "Text at the bottom of the review screen encouraging the " +
-                                            "user to check sharpness by double-tapping the image")
-    
+    @objc public var reviewTextBottom: String = .localized(resource: ReviewStrings.bottomText)
```

Comment:
I suppose that there are two equivalent ways to call these methods:

`reviewTextBottom: String = .localized` and
`reviewTextBottom = String.localized`

In my opinion the second is a bit better. But maybe just my preference.

Reply:
It could be better for those situations when you are forced to add it to the property to resolve the ambiguity, but there are a lot of different situations where it is not needed. For example, when assigning an already defined `String` property (i.e: `title` in a `UIViewController`)